src/main/java/com/syncstream_dev/syncstream/model/VideoInfo.java

```diff
@@ -0,0 +1,25 @@
+package com.syncstream_dev.syncstream.model;
+
+public class VideoInfo {
+    private String url;
+    private String title;
+    private Long timestamp;
+
+    public VideoInfo(String url, String title, Long timestamp)  {
```

Comment:
Setters may be beneficial for future development. (e.g. User switches video, all of `url`, `title` and `timestamp` will have to change)

Reply:
1a47af35ac9d322df9e9989e40623249abd61f78 fixes this